GetBlogFeeds.cs

```diff
@@ -129,7 +129,7 @@ protected override async Task<IEnumerable<IDocument>> ExecuteInputAsync(IDocumen
                 }
             }
 
-            return null;
+            return input.Yield();
```

Comment:
This is what caused the bug in #390 - we need to return the original document if there's no RSS feed or if something went wrong getting the feed.